gsa/src/gmp/models/task.js

```diff
@@ -87,7 +87,7 @@ export function parse_yes(value) {
   return value === 'yes' ? YES_VALUE : NO_VALUE;
 }
 
-const parseIntoArray = value => (isArray(value) ? value : [value]);
+export const parseIntoArray = value => (isArray(value) ? value : [value]);
```

Comment:
Could you extract this function in to gmp/parser, please? It should not be exported by one model to use in another. We'd have an unnecessary dependency between task and audit model.